### ISSUE TITLE
Implement @font-face injection

### ISSUE THREAD

**Issue by roman01la** [roman01la/cljss#27]
`font-face` macro should construct CSS `@font-face` at-rule and expand into a function which injects declaration at runtime

Support all descriptors as defined here https://developer.mozilla.org/en-US/docs/Web/CSS/%40font-face

Should be something like
```clojure
(require '[cljss.core :refer [font-face]])

(font-face {:font-family "Patrick Hand SC"
            :font-style "normal"
            :font-weight 400
            :src [{:local "Patrick Hand SC"}
                  {:local "PatrickHandSC-Regular"}
                  {:url "https://fonts.gstatic.com/s/patrickhandsc/v4/OYFWCgfCR-7uHIovjUZXsZ71Uis0Qeb9Gqo8IZV7ckE.woff2"
                   :format "woff2"}
                  {:url "https://fonts.gstatic.com/s/patrickhandsc/v4/OYFWCgfCR-7uHIovjUZXsZ71Uis0Qeb9Gqo8IZV7ckE.otf"
                   :format "opentype"}]
            :unicode-range ["U+0100-024F" "U+1E00-1EFF"]})
```

**Comment by roman01la** [comment]
fixed in `1.5.9`

**Comment by Atsman** [comment]
I like that at least font-face at-rule you decided to implement as macro. I hope that media at-rule will be implemented in similar way :)
